cidr_func/src/lib.rs

```diff
@@ -14,7 +14,11 @@ pub fn netmask(ip: &str) -> Result<String, String> {
 #[udf]
 pub fn base_ip(ip: &str) -> Result<String, String> {
     match ip.parse::<IpNetwork>() {
+<<<<<<< Updated upstream
         Ok(network) => Ok(network.ip().to_string()),
+=======
+        Ok(network) => Ok(network.network().to_string()),
+>>>>>>> Stashed changes
```

Comment:
Is the PR not correctly merged?

Reply:
something wrong when I do the rebase, fixed it